docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -6,9 +6,9 @@
 EA App currently displays a blank screen when using NorthstarProton, however Northstar will still launch assuming you have logged in to EA App at least once.
 {% endhint %}
 
-> **Check your GLIBC version.** NorthstarProton currently only supports version 2.33 and higher. Verify your installed version with `ldd --version`. **Ubuntu 20.04 LTS**, **Debian 11**, and **Void Linux** are known to have outdated GLIBC packages, meaning you will have to experiment with finding a Proton version that works for you, or use Lutris. It is therefore strongly recommended to use an up-to-date Linux distro to be able to make use of NorthstarProton. This check does not need to be completed on Steam Deck or Steam OS.
+> **Northstar does not work on Ubuntu and Debian distros.** The workaround to this is using Distrobox, which will be explained further down, or using a known supported distro like Fedora or Arch. You will still need to follow the instructions to install Northstar with the Native Package path.
 
-On Steam Deck, complete the following in desktop mode. You may return to game mode once completed _(A mouse + keyboard plugged into the Deck are recommended for easier navigation of menus)_
+If you are using a Steam Deck, complete the following in desktop mode. You may return to game mode once completed _(A mouse + keyboard plugged into the Deck are recommended for easier navigation of menus)_
```

Comment:
Previous was fine or what is the reason for change here? ^^

Reply:
changed it to make it more clear that the instructions aren't *only* for steam deck since it could be interpreted that way (and also has been in the past)